src/ResourceManager/Resources/Commands.Resources/RoleDefinitions/GetAzureRoleDefinitionCommand.cs

```diff
@@ -48,14 +48,17 @@ public class GetAzureRoleDefinitionCommand : ResourcesBaseCmdlet
 
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, ParameterSetName = ParameterSet.RoleDefinitionName, HelpMessage = "If specified, displays the the roles at and below scope.")]
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, ParameterSetName = ParameterSet.RoleDefinitionCustom, HelpMessage = "If specified, displays the the roles at and below scope.")]
+        [Obsolete("GEt-AzureRmRoleDefinition: The parameter \"AtScopeAndBelow\" is being removed in an upcoming breaking change release.")]
```

Comment:
change "GEt" to "Get".  Is there any migration information for this, or should users just stop using this parameter?

Reply:
this filter isnt supported in the service,in earlier versions it was being ignored.so to make it consistent with the service,we are removing it in the breaking change release